Documentation/sources/content-card-ui/public-classes/Styles/smallimageuistyle.md

```diff
@@ -0,0 +1,36 @@
+# Class - SmallImageUIStyle
+
+Class representing the style for a small image AEP UI.
+
+## Public Properties
```

Comment:
Do you think its useful to add a diagram here showing the layout of small image card and which area each style property affects? I have one here which you can use
https://wiki.corp.adobe.com/display/adms/Using+Content+Cards+with+UI+on+Android#UsingContentCardswithUIonAndroid-StylingContentCards